src/Network/Network/help/New-AzVpnSiteLink.md

```diff
@@ -0,0 +1,162 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version:
```

Comment:
Missing url for online docs, should be `https://docs.microsoft.com/en-us/powershell/module/az.network/new-azvpnsitelink`

Reply:
updated. shouldn't this be auto generated?